### PR TITLE
Add goodness of fit metric

### PR DESCRIPTION
**Pull request by CeliaBenquet**
- Add the goodness of fit computation function and wrapper function to be used directly on a `cebra.CEBRA` model (sklearn implementation only). The goodness of fit computation function can be applied on `solver.history` (loss) for the pytorch implementation, and in that case the batch size needs to be provided.
- Add corresponding tests.

Note that it is not implemented for batch size set to `None`.

### CODE REVIEW

**Review by stes — COMMENTED** [review]
Duplicate of #190, just without tests?

**Comment by CeliaBenquet** [comment]
Ah yeah right, didn't see that @stes. Happy to review when it's done then.
